### ISSUE TITLE
super broken in npm

### ISSUE THREAD

**Issue by stephenmathieson** [radubrehar/react-accordion#3]
you just name squatting or something?

**Comment by radubrehar** [comment]
Nope. I already have an implementation, and will fix it & launch an update for the latest React version in a month or so.
